### PR TITLE
Ability to generate a random name with some prefix for the created migration

### PR DESCRIPTION
**Pull request by zakonnic**
By default user is prompted to enter a migration name if it's not specified by the parameter. If it is difficult to come up with unique names for new migrations, or we have plenty of them, it's more convenient to generate a random name and offer it by default. If the option "default_migration_prefix" is specified in the config, a name will be offered starting with this line and ending with a random hash. Otherwise, current behavior will not change.

### CODE REVIEW

File: src/Migration/Generator/MigrationGenerator.php [hunk 1]
```diff
@@ -140,7 +154,8 @@ public function generate(): int
         }
 
         if (empty($this->settings['name'])) {
-            $name = $this->io->ask('Enter migration name', '');
+            $defaultMigrationName = $this->generateDefaultMigrationName();
```
Comment (odan) [inline src/Migration/Generator/MigrationGenerator.php]: The $defaultMigrationName  should only be used if `default_migration_prefix` is not empty.

**Comment by odan** [comment]
Please resolve this conflict before I can merge it. Thanks!

**Comment by odan** [comment]
Thank you very much!

**Comment by zakonnic** [comment]
Thank you, @odan!

**Pull request merged**